webpack/scenes/Content/ContentConfig.js

```diff
@@ -81,6 +82,7 @@ export default [
       singularLowercase: __('OSTree ref'),
       pluralLabel: 'ostree_refs',
       singularLabel: 'ostree_ref',
+      capsuleCountLabel: 'ostree_refs',
```

Comment:
```suggestion
      capsuleCountLabel: 'ostree_ref',
```

Than ostree should show up!

![image](https://github.com/Katello/katello/assets/14796566/595b9f65-b517-4e55-9d77-fd1129329ee5)